### PR TITLE
Implemented `no-todo` code style rule.

### PR DESCRIPTION
**Pull request by arturocuya**

### CODE REVIEW

File: src/plugins/codeStyle/index.ts [hunk 1]
```diff
@@ -85,6 +86,14 @@ export default class CodeStyle {
                 if (validateAAStyle) {
                     this.validateAAStyle(e, aaCommaStyle, diagnostics);
                 }
+            },
+            CommentStatement: e => {
+                if (validateTodo) {
+                    console.log('this.lintContext', this.lintContext);
```
Comment (philippe-elsass-deltatre) [inline src/plugins/codeStyle/index.ts:92]: No logs

File: src/util.ts [hunk 2]
```diff
@@ -97,6 +97,8 @@ function tryLoadConfig(filename: string): BsLintConfig | undefined {
 export interface PluginContext {
     program: Readonly<Program>;
     severity: Readonly<BsLintRules>;
+    // TODO: Is this the right place to put this?
```
Comment (philippe-elsass-deltatre) [inline src/util.ts:100]: Yes! Though I'm not sure how well it will scale as rules are added...

**Review by philippe-elsass-deltatre — COMMENTED** [review]
Lovely addition!

**Comment by arturocuya** [comment]
@philippe-elsass-deltatre I just updated the code to remove the `console.log` and the TODO comment :)

**Review by elsassph — APPROVED** [review]

**Pull request merged**